test.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+import winnowing
+
+
+class TestWinnowing(unittest.TestCase):
+    # testing for R language
+    def test1(self):
+        check = winnowing.Winnowing(file1= "./tests/test6.r",file2="./tests/test7.r",lang="r")
```

Comment:
Create a `setup` function in the class. Use  `os` module to get the file path of the test files in the setup function. **You should never add paths as strings**. Apply wherever necessary.